### DISCUSSION TITLE
Mapping Nuclei Templates to MITRE ATT&CK

### DISCUSSION THREAD

**Discussion by AsrithaNarina** [projectdiscovery/nuclei-templates#4673] [Ideas]
As we all know https://attack.mitre.org/ is now playing an prominent role in mapping Tactics-(sub)Techniques with the threat actors.
Also we can map malicious behaviors to this framework. I have an idea of mapping these nuclei templates to Tactics-(sub)Techniques which can add more value to derive security flaws in software.

Let me know what do you think!!!!

**Comment by ehsandeep** [comment]
@AsrithaNarina currently we are mapping [known exploited vulnerabilities catalog](https://www.cisa.gov/known-exploited-vulnerabilities-catalog) list by CISA with templates using tags and will be automated soon as detailed here - https://github.com/projectdiscovery/nuclei-templates/issues/4719, similarly do you have idea or could share details around what mapping we can do with template when it comes to MITRE ATT&CK framework?

**Comment by MetzinAround** [comment]
Closing older Issue. :)
